coverage report in workflow are not created

Expected results:
coverage report are created

Actual Results:
`Run coverage run -m pytest
ImportError while loading conftest '/home/runner/work/quiz-roads/quiz-roads/tests/conftest.py'.
tests/conftest.py:10: in <module>
    from quizproject.models.users import User
E   ModuleNotFoundError: No module named 'quizproject.models.users'
Error: Process completed with exit code 4.`

references #31

--- a/readme.md
+++ b/readme.md
@@ -1,6 +1,6 @@
 #  Quiz - Roads 
 [![Build](https://github.com/DarekRepos/quiz-roads/actions/workflows/build.yml/badge.svg)](https://github.com/DarekRepos/quiz-roads/actions/workflows/build.yml)
-[![Tests Status](./reports/junit/junit-badge.svg?dummy=8484744)](./reports/junit/report.html)
+[![Tests Status](https://github.com/DarekRepos/quiz-roads/blob/badges/coverage/coverage-unit-badge.svg?dummy=8484744)](./reports/junit/report.html)
 [![Coverage Status](https://github.com/DarekRepos/quiz-roads/blob/badges/coverage/coverage-badge.svg?dummy=8484744)](./reports/coverage/index.html)
 [![Flake8 Status](https://github.com/DarekRepos/quiz-roads/blob/badges/flake8/flake8-badge.svg?dummy=8484744)](./reports/flake8/index.html)
 

